narwhal/consensus/src/dag.rs

```diff
@@ -102,12 +101,13 @@ enum DagCommand {
 
 impl InnerDag {
     fn new(
-        committee: &Committee,
+        _committee: &Committee,
         rx_primary: mpsc::Receiver<Certificate>,
         rx_commands: Receiver<DagCommand>,
         dag: NodeDag<Certificate>,
         vertices: RwLock<BTreeMap<(PublicKey, Round), CertificateDigest>>,
         rx_shutdown: ConditionalBroadcastReceiver,
+        genesis_certs: Vec<Certificate>,
```

Comment:
can't we obtain those from the `Committee` anymore?

Reply:
We can but it requires the private key and I opted not to pass that in to every object. Instead the certs are generated at the top level and passed in.

Reply:
then I'm guessing we can remove the `_committee` param, judging by the underscore prefix?

Reply:
Yes here too, rust-analyzer is confused about this in many places.